sq/tests/sdkms.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash -e
```

Comment:
Can you set up CI to run this automatically?

Reply:
That is the idea, Cuneyt is looking into it.